### PR TITLE
Reworded warning for clarity

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

As per issue #6586, updated the note to specify that the conflict with Entra ID only applies to backoffice logins, not member logins. Added a line about the upcoming feature (WIP) which might remove this limitation. 

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [x] Other

## Product & version (if relevant)

CMS v14.

## Deadline (if relevant)

Anytime

### CODE REVIEW

File: 14/umbraco-cms/tutorials/add-microsoft-entra-id-authentication.md [hunk 1]
```diff
@@ -9,11 +9,13 @@ description: >-
 This tutorial takes you through configuring Microsoft Entra ID (Azure Active Directory/Azure AD) for the member login on your Umbraco CMS website.
 
 {% hint style="warning" %}
-Entra ID conflicts with Umbraco ID which is the main authentication method used on all Umbraco Cloud projects.
+Entra ID (formerly Azure AD) conflicts with Umbraco ID when used for backoffice logins on Umbraco Cloud projects, though this does not affect member logins.
 
-Due to this, we **highly recommend not using Azure AD for backoffice authentication on your Umbraco Cloud projects**.
+**We highly recommend not using Azure AD for backoffice authentication on your Umbraco Cloud projects**.
```
Comment (eshanrnh) [inline 14/umbraco-cms/tutorials/add-microsoft-entra-id-authentication.md:14]:
```suggestion
**We strongly recommend against using Entra ID for backoffice authentication on Umbraco Cloud projects**.
```

File: 14/umbraco-cms/tutorials/add-microsoft-entra-id-authentication.md [hunk 2]
```diff
@@ -9,11 +9,13 @@ description: >-
 This tutorial takes you through configuring Microsoft Entra ID (Azure Active Directory/Azure AD) for the member login on your Umbraco CMS website.
 
 {% hint style="warning" %}
-Entra ID conflicts with Umbraco ID which is the main authentication method used on all Umbraco Cloud projects.
+Entra ID (formerly Azure AD) conflicts with Umbraco ID when used for backoffice logins on Umbraco Cloud projects, though this does not affect member logins.
```
Comment (eshanrnh) [inline 14/umbraco-cms/tutorials/add-microsoft-entra-id-authentication.md:12]:
```suggestion
Entra ID (formerly Azure AD) conflicts with Umbraco ID when used for backoffice user logins on Umbraco Cloud projects, though this does not affect member logins.
```

File: 14/umbraco-cms/tutorials/add-microsoft-entra-id-authentication.md [hunk 3]
```diff
@@ -9,11 +9,13 @@ description: >-
 This tutorial takes you through configuring Microsoft Entra ID (Azure Active Directory/Azure AD) for the member login on your Umbraco CMS website.
 
 {% hint style="warning" %}
-Entra ID conflicts with Umbraco ID which is the main authentication method used on all Umbraco Cloud projects.
+Entra ID (formerly Azure AD) conflicts with Umbraco ID when used for backoffice user logins on Umbraco Cloud projects, though this does not affect member logins.
```
Comment (eshanrnh) [inline 14/umbraco-cms/tutorials/add-microsoft-entra-id-authentication.md:12]:
```suggestion
Entra ID (formerly Azure Active Directory) conflicts with Umbraco ID when used for backoffice user logins on Umbraco Cloud projects, though this does not affect member logins.
```

File: 14/umbraco-cms/tutorials/add-microsoft-entra-id-authentication.md [hunk 4]
```diff
@@ -9,11 +9,13 @@ description: >-
 This tutorial takes you through configuring Microsoft Entra ID (Azure Active Directory/Azure AD) for the member login on your Umbraco CMS website.
 
 {% hint style="warning" %}
-Entra ID conflicts with Umbraco ID which is the main authentication method used on all Umbraco Cloud projects.
+Microsoft Entra ID (formerly Azure Active Directory) conflicts with Umbraco ID when used for backoffice user logins on Umbraco Cloud projects, though this does not affect member logins.
```
Comment (eshanrnh) [inline 14/umbraco-cms/tutorials/add-microsoft-entra-id-authentication.md:12]:
```suggestion
Microsoft Entra ID (formerly Azure Active Directory) conflicts with Umbraco ID when used for backoffice user logins on Umbraco Cloud projects. This does not affect member logins.
```

File: 14/umbraco-cms/tutorials/add-microsoft-entra-id-authentication.md [hunk 5]
```diff
@@ -9,11 +9,13 @@ description: >-
 This tutorial takes you through configuring Microsoft Entra ID (Azure Active Directory/Azure AD) for the member login on your Umbraco CMS website.
 
 {% hint style="warning" %}
-Entra ID conflicts with Umbraco ID which is the main authentication method used on all Umbraco Cloud projects.
+Microsoft Entra ID (formerly Azure Active Directory) conflicts with Umbraco ID when used for backoffice user logins on Umbraco Cloud projects. This does not affect member logins.
```
Comment (eshanrnh) [inline 14/umbraco-cms/tutorials/add-microsoft-entra-id-authentication.md:12]:
```suggestion
Microsoft Entra ID conflicts with Umbraco ID when used for backoffice user logins on Umbraco Cloud projects. This does not affect member logins.
```

**Review by sofietoft — COMMENTED** [review]
Looks great @eshanrnh !

I'm not sure about the note about the upcoming feature on Cloud.
It's a temporary note that we will somehow need to keep track of, to make sure we remove it once the feature is actually launched 🤔

File: 14/umbraco-cms/tutorials/add-microsoft-entra-id-authentication.md [hunk 6]
```diff
@@ -9,11 +9,13 @@ description: >-
...
+Microsoft Entra ID conflicts with Umbraco ID when used for backoffice user logins on Umbraco Cloud projects. This does not affect member logins.
 
-Due to this, we **highly recommend not using Azure AD for backoffice authentication on your Umbraco Cloud projects**.
+**We strongly recommend against using Entra ID for backoffice authentication on Umbraco Cloud projects**.
 
-It is still possible to use other [External Login Providers](../reference/security/external-login-providers.md) like Google Auth and OpenIdConnect, with Umbraco Cloud.
+For member authentication, other [External Login Providers](../reference/security/external-login-providers.md), such as Google Auth and OpenIdConnect, are still supported and compatible with Umbraco Cloud.
+
+Additionally, the **Custom Identity Provider for Umbraco Cloud** feature is currently in development. Once released, it will enable seamless integration of your own Entra ID with the Umbraco Cloud Portal, providing enhanced user authentication. For more information, see the [Umbraco Roadmap](https://umbraco.com/products/knowledge-center/roadmap/).
```
Comment (eshanrnh) [inline 14/umbraco-cms/tutorials/add-microsoft-entra-id-authentication.md:18]:
```suggestion
```

**Comment by eshanrnh** [comment]
I removed the line about the upcoming feature but I believe even the Note will have to be removed once the feature is released since then Entra ID will be compatible with backoffice users as well.

**Comment by sofietoft** [comment]
> I removed the line about the upcoming feature but I believe even the Note will have to be removed once the feature is released since then Entra ID will be compatible with backoffice users as well.

Yes, that's a good point!
Let's talk about how we can keep track of this somehow, so we remember to remove it once the feature is out 💪

**Review by sofietoft — APPROVED** [review]

**Pull request merged**